content/en/docs/release_notes/villasimius.md

```diff
@@ -12,7 +12,8 @@ Brief summary of the notable release contents.
 
 <!-- ## Breaking changes -->
 
-<!-- ## Features -->
+## Features
+* #7406: When installing plugins from the **Plugins** page, if the latest version of a plugin is not compatible with the current version of {{% variables/apibuilder_prod_name %}} or Node.js a previous compatible version will be installed instead, if possible.
```

Comment:
Add note: this will only try to install a compatible version when using Node.JS 12.13 and above.